tests/unit/amo/components/TestCollection.js

```diff
@@ -748,9 +735,7 @@ describe(__filename, () => {
     });
 
     const editLink = wrapper.find('.Collection-edit-link').find(Button);
-    expect(editLink).toHaveLength(1);
-    expect(editLink).toHaveProp('to',
-      `/collections/${defaultUser}/${defaultCollectionDetail.slug}/edit/?page=${page}`);
+    expect(editLink).toHaveLength(0);
```

Comment:
As above, this test just needs to be changed to use a user who owns the collection.